mosql.gemspec

```diff
@@ -24,9 +24,9 @@ Gem::Specification.new do |gem|
 
   gem.add_runtime_dependency "mongoriver", "0.4"
 
-  gem.add_runtime_dependency "mongo", "~> 1.10"
-  gem.add_runtime_dependency "bson", "~> 1.10"
-  gem.add_runtime_dependency "bson_ext", "~> 1.10"
+  gem.add_runtime_dependency "mongo", "~> 1.0"
+  gem.add_runtime_dependency "bson", "~> 1.0"
+  gem.add_runtime_dependency "bson_ext", "~> 1.0"
```

Comment:
Don't change dependencies like this. Under 1.9 had a lot of bugs.

Your eventual aim should be a pull request back to mosql mainline. They wouldn't accept lowering the minimum version.